cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -83,7 +83,7 @@ func kubeDNSAddon(cfg *kubeadmapi.MasterConfiguration, client clientset.Interfac
 			DNSBindAddr:    dnsBindAddr,
 			DNSProbeAddr:   dnsProbeAddr,
 			DNSDomain:      cfg.Networking.DNSDomain,
-			DNSProbeType:   GetKubeDNSProbeType(k8sVersion),
```

Comment:
I'd just build this into the manifest again; and not have it at all as a gotmpl parameter